core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -107,6 +109,21 @@
         return groupNames;
     }
 
+    public Set<String> getOozieGroups(Stack stack) {
+        LOGGER.debug("Fetching list of instance groups with Oozie installed");
+        Cluster cluster = stack.getCluster();
+        if (cluster != null) {
+            Set<String> groupNames;
```

Comment:
Why do we need for the early definition of this variable? You could just do `Set<String> groupNames = cmTemplateProcessor ...` below.